CONTRIBUTING.md

```diff
@@ -18,65 +18,52 @@ The primary aim of the OWASP Application Security Verification Standard (ASVS) P
-
-If you are comfortable that your query has  has not been previously discussed, you can open an issue. Please try and include the ASVS text you are talking about in the issue (or at least the value of the "#v5.0.be" column) to save having to jump back and forth and please carry out all discussion in the associated issue and not in a PR discussion.
-
-### How to suggest changes to the release candidate during this period
+### Numbering changes compared to the issues.
 
-Note that review and changes should always be made based on the raw .md files. The other output formats have not yet been updated.
+Note that since there were major numbering changes between v4.0.3 and the bleeding edge version of v5.0.0 (known as v5.0.be), and between v5.0.be and the final v5.0.0 release. As such, a lot of the requirement numbers in the issues may be different, depending on when the discussion was taking place. The following rough guide to numbering applies:
 
-To help those who are using the "bleeding edge" version for their tests and in order to be able to track changes made during this period, we have prepared a set of labels to be used when making changes to the "bleeding edge" version during this time. The current bleeding edge working directory can be found here <https://github.com/OWASP/ASVS/tree/master/5.0/en>.
+* The v4.0.3 numberings were valid at the time of its release in October 2021.
+* The v5.0.be numberings cover the period from then until the major renumbering around 27th March 2025
+* The v5.0.0 numberings cover the period from the major renumbering until the v5.0.0 release at the end of May 2025. 
 
-### Mapping between v4.0.3 and v5.0.0
+The mappings in the [mappings folder](https://github.com/OWASP/ASVS/tree/master/5.0/mappings/) can be used to check what number the requirement was at the point in time it was being discussed:
 
-For a smooth transfer from ASVS v4.0.3 to v5.0.0, it is good to know:
+If you are comfortable that your query has  has not been previously discussed, you can open an issue. Please try and include the ASVS requirement number and text you are talking about in the issue to save having to jump back and forth and please carry out all discussion in the associated issue and not in a PR discussion.
 
-  * What happened with a particular requirement since v4.0.3?
-  * Does a requirement in v5.0.0 originate from requirement/s from v4.0.3, or is it completely new?
-
-Through the 3.5 years since the last release (v4.0.3), changes have been tracked and tagged to make it possible to provide a 2-way mapping now.
-
-(Note that requirement numbers for v5.0.0 may change until it is released. Although we are continuing to update the mapping as we go. It is therefore too early to make "a final copy" of the mapping.)
-
-Mappings are presented in separate `yml` files. However, a formatted output is also provided on <asvs.dev>:
+<!-- DO WE STILL WANT THIS HERE???????
```

Comment:
We need to decide if and how to include this

Reply:
How do to post-release contributing requires some discussion anyway